doc/crypto/overview/functionality.rst

```diff
@@ -114,15 +114,21 @@ Cryptographic operations
 
 For each type of cryptographic operation, the API can include:
 
-*   One or more *single-part* functions, which perform a whole operation in a single function call. For example, compute, verify, encrypt or decrypt.
-*   A *multi-part operation* --- which is a series of functions that work with a stored operation state.
+*   One or more *single-part* functions, which perform a whole operation in a single function call. For example, compute, verify, encrypt or decrypt. See :secref:`single-part-functions`.
+
+*   A *multi-part operation* --- which is a set of functions that work with a stored operation state. This provides more control over operation configuration, piecewise processing of large input data, or handling for multi-step processes. See :secref:`multi-part-operations`.
```

Comment:
Preexisting: it's weird that “one or more single-part functions” is put in parallel with “a multi-part operation”. Each bullet point makes sense on its own, they're just weird when contrasted. Here's my attempt at polishing this.

> The API supports cryptographic operations through two kinds of interfaces:
>
> * A *single-part* function performs a whole operation in a single function call. For example, compute, verify, encrypt or decrypt. See :secref:`single-part-functions`.
> 
> *   A *multi-part operation* is a set of functions that work with a stored operation state. This provides more control over operation configuration, piecewise processing of large input data, or handling for multi-step processes. See :secref:`multi-part-operations`.
>
> Depending on the mechanism, one or both kind of interfaces may be provided.

Reply:
I like your proposed modification, and will adopt it.